### PR TITLE
Add jwt validation

### PR DESCRIPTION
**Pull request by mackosx**
Used https://github.com/panva/jose/blob/main/docs/functions/jwks_remote.createRemoteJWKSet.md as a reference.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,57 +1,15 @@
...
-
-    options = options || {};
-    var pos = options.header === true ? 0 : 1;
-    try {
-      return JSON.parse(base64UrlDecode(token.split('.')[pos]));
-    } catch (e) {
-      throw new InvalidTokenError('Invalid token specified: ' + e.message);
-    }
+    const { payload } = await jwtVerify(token, JWKS);
+    return payload;
```
Comment (ryanluker) [inline app.js:14]: I wonder if we can print to the console or the html dom some info about the verification 🤔.

**Review by ryanluker — APPROVED** [review]
Provided one thought, but otherwise looking great!
Should be fine to also convert the gs-oidc-client as well to use this.

File: app.js (hunk 1 above)
Comment (mackosx) [reply app.js:14]: Ooh yeah that would be good. Incoming...

Comment (mackosx) [reply app.js:14]: Like this:
image

**Review by ryanluker — APPROVED** [review]

**Pull request merged**